src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java

```diff
@@ -3,26 +3,33 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
```

Comment:
the wildcard import approach has advantages, but its usage can cause problems in certain situations

please turn off Wildcard imports (https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-)